src/main/scala/com/datastax/gatling/plugin/model/DseCqlAttributes.scala

```diff
@@ -21,6 +21,7 @@ import io.gatling.core.check.Check
   * @param tag              Name of Query to include in reports
   * @param statement        CQL Statement to be sent to Cluster
   * @param cl               Consistency Level to be used
+  * @param dynamicCl         Consistency Level to be used, will be called on every request
```

Comment:
It looks like there is a whitespace issue 

```suggestion
  * @param dynamicCl        Consistency Level to be used, will be called on every request
```